eventually/utils/jwttoken.py

```diff
@@ -0,0 +1,57 @@
+"""
+JWT Token
+=========
+
+The module that provides creating and handling JWT tokens.
+"""
+
+import jwt
+from django.conf import settings
+from django.utils import timezone
+
+SECRET_KEY = settings.JWT_TOKEN_KEY
+ALGORITHM = settings.JWT_ALGORITHM
+
```

Comment:
if you wrote JWT_TOKEN_KEY and JWT_ALGORITHM in settings why you didn't add this file?